### ISSUE TITLE
pkg: sqlite error

### ISSUE THREAD

**Issue by tomasantunes** [ravynsoft/ravynos#306]
I downloaded 0.4.0pre4 and while I was following the install instructions on the wiki I had an error while executing this command:

```
/usr/sbin/pkg remove -y furybsd-live-settings freebsd-installer
```

The error is:
```
pkg: warning: database version 36 is newer than libpkg(3) version 35, but still compatible
pkg: sqlite error while executing INSERT OR IGNORE INTO mtree(content) VALUES(?1) in file pkgdb.c:115: no suck table: mtree
```

**Comment by mszoek** [comment]
Hmm, thanks for the report. Will look into that!

**Comment by linz2h** [comment]
Just `rm -rf /var/db/pkg` then pkg update. It works for me

**Comment by tomasantunes** [comment]
Yes, that removes the packages. It doesn't update because it's not connected to the internet.

**Comment by linz2h** [comment]
I think there's way to install pkg locally, but you'd better update through offical repo, make it easier

**Comment by mszoek** [comment]
Fixed in 0.5.0
